packages/backend/src/database/entities/spaces.ts

```diff
@@ -204,6 +209,9 @@ export const getSpaceWithQueries = async (
                 savedQuery.chart_config,
             ),
             views: parseInt(savedQuery.views, 10) || 0,
+            firstViewedAt: savedQuery.first_viewed_at
+                ? new Date(savedQuery.first_viewed_at).toJSON()
```

Comment:
I don't think we convert dates to JSON , it should be handled automatically... why do you need it here ?

Reply:
even if you want to convert date into string, I'd suggest you use `moment`

Reply:
so.. this function returns Space which is used in a lot of places on the frontend side. saying that `firstViewedAt` is wrong because once it is serialised it will be converted to a string. and dates are serialised using toJSON(). I am just trying to remove a confusion on the frontend side where the type says it's a date but in reality it is string. do you have any better ideas?